tests/test_fileio/test_data_source.py

```diff
@@ -36,26 +35,25 @@ def test_to_yaml(self):
         """
         value = data_source.Experiment('My First Experiment',
                                        'A Lab Instrument',
-                                       datetime(1992, 7, 14, 10, 10,
-                                                10), 'x-rays')
+                                       datetime(1992, 7, 14).strftime("%Y-%m-%d"), 'x-rays')
         assert value.to_yaml() == 'title: My First Experiment\n'\
-            + 'instrument: A Lab Instrument\ndate: 1992-07-14T'\
-            + '10:10:10\nprobe: x-rays\n'
+            + 'instrument: A Lab Instrument\nstart_date: \'1992-07-14\''\
```

Comment:
IMO the comparison of a string with the output of a `to_yaml` is brittle. If the files validate, and we can check individual entries are correct, then all this test ensures is that the yaml package produces yaml.

Reply:
Suggestions on how we improve?

Reply:
I'm not sure what the test is examining. It's basically seeing that pyyaml is emitting yaml.

Reply:
Is your suggestion then that it is removed?

Reply:
Can we validate against the schema?